### PR TITLE
Added a dashed type button

### PR DESCRIPTION
**Pull request by jamieleo93**
I have added a dashed style button type

Issue: Add New Buttons

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -235,6 +235,12 @@ $(document).ready(function () {
       ],
       children: [],
     },
+    {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:238]: Please add this button type in the correct alphabetical order (at the time of writing this, under Animated)

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @jamieleo93 it looks great. My only remark is that the white button has no dashed border, can you fix that?

File: assets/js/buttons-examples.js (hunk 1 above)
Comment (jamieleo93) [reply assets/js/buttons-examples.js:238]: Yup I'll do that just now - sorry for missing that out!

**Comment by jamieleo93** [comment]
@shahednasser I've made the changes - let me know if you need anything else updated

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @jamieleo93 for code.

**Pull request merged**